### DISCUSSION TITLE
Calendar Size Modifications

### DISCUSSION THREAD

**Discussion by boncz92** [EvotecIT/PSWriteHTML#320] [Q&A] [answered]
Is there any way currently to modify the content size of the calanders? I am attempting to use this to automatically generate a calendar of scheduled tasks across multiple servers and would be better served if I could increase the height at which it renders

I have plans to categorize the tasks, but still even with the categorization there will be an abundance of entries. I know the timline view is only usable by using the premium calendar plugin, so this is my second best choice.

![unknown](https://user-images.githubusercontent.com/8141438/170418175-c7571abe-0173-4f5d-b99f-fcb9f1ac1656.png)
.

**Comment by PrzemyslawKlys** [comment, marked as answer]
It's not part of the PSWriteHTML, but it's doable as far as I can see:
- https://fullcalendar.io/docs/contentHeight
- https://fullcalendar.io/docs/v4/eventLimit

I guess what you could do is use tabs and multiple calendars or maybe utilize custom button:
- https://fullcalendar.io/docs/toolbar 

And create some kind of choice of events to show.
